pages/common/firebase.md

```diff
@@ -1,25 +1,28 @@
 # firebase
 
-> Firebase Command Line Interface (CLI) Tools.
-> Used to test, manage, and deploy your Firebase project from the command line.
+> Test, manage, and deploy Firebase projects from the command line.
 > More information: <https://github.com/firebase/firebase-tools>.
 
-- Login:
+- Login to <https://console.firebase.google.com>:
```

Comment:
can the user login to another server?

Reply:
I don't think so, but I'm not sure.  I'll check later

Reply:
Ping @bl-ue

Reply:
@navarroaxel Nope, this is the only server users can login to.  But I think we should still leave it in because it's probable that people will want to know.  I would, at least.